glide.lock

```diff
@@ -178,7 +178,7 @@ imports:
   version: 5bd2802263f21d8788851d5305584c82a5c75d7e
 - name: github.com/satori/go.uuid
   version: b061729afc07e77a8aa4fad0a2fd840958f1942a
-- name: github.com/Sirupsen/logrus
+- name: github.com/sirupsen/logrus
```

Comment:
Did you regenerate this lock file?  I assume not as the hash ID has not changed.